integration-cli/docker_cli_build_test.go

```diff
@@ -7357,3 +7357,19 @@ func (s *DockerSuite) TestBuildWindowsEnvCaseInsensitive(c *check.C) {
 		c.Fatalf("Case insensitive environment variables on Windows failed. Got %s", res)
 	}
 }
+
+func (s *DockerSuite) TestBuildLabelsArgs(c *check.C) {
```

Comment:
Thanks @AkihiroSuda. In #29619 the issue was only triggered by `docker build --label A=A --label B=B ...` through command line. `LABEL` inside Dockerfile directly will not trigger the issue. So I think it probably make sense to have a IT for it.